packages/luna/src/components/Chat/ChatBubble/ChatBubble.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { ChatBubble, ChatBubbleVariant } from "@/components/Chat/ChatBubble";
+import { DarkStoryWrapper } from "@/internal/DarkWrapper";
+import { faker } from "@faker-js/faker/locale/ar";
+import React from "react";
+
+type Component = typeof ChatBubble;
+
+const meta: Meta<Component> = {
+  component: ChatBubble,
+  parameters: { layout: "centered" },
+  decorators: [
+    (Story) => {
+      return (
+        <div className="tw-max-w-[352px]">
+          <Story />
+        </div>
+      );
+    },
+    DarkStoryWrapper,
+  ],
+};
+
+export const Primary: StoryObj<Component> = {
+  args: {
+    text: faker.lorem.words(10),
+    variant: ChatBubbleVariant.CurrentUser,
```

Comment:
It is better to use union over enums. We are moving toward unions and will soon omit most of our enums.